### PR TITLE
Max/move add task button

### PR DESCRIPTION
**Pull request by maxwolffe**
Make app compatible with Android Studio 1.0
Move the add task button to only appear in the task list view. (required adding menu inflaters to all fragments, but I think that's ok. because we'll want to add creation buttons to some of the fragments anyway?)
Also, app is running out of memory, but I don't think that's an issue with this PR.

### CODE REVIEW

**Comment by negativetwelve** [comment]
:heart:

**Comment by maxwolffe** [comment]
hi hi hi hi hi. 

@nishdesai  I think @charlesx2013  wants this merged so he can work on task editing. :heart:

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Sites/CreateSiteRequest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import java.util.HashMap;
+
+/**
+ * Created by maxwolffe on 2/16/15.
+ */
+public class CreateSiteRequest extends BaseRequest {
+    Site mSite;
+    Activity mActivity;
+
+    public CreateSiteRequest(final Activity activity, final Site site, HashMap<String, JSONObject> params, final Response.Listener<Site> listener) {
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Sites/CreateSiteRequest.java]: This should have a Javadoc.

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Sites/CreateSiteRequest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                            Log.e("Json exception", e.toString());
+                        }
+                    }
+                }, activity);
+
+        mSite = site;
+        mActivity = activity;
+    }
+
+    protected static JSONObject siteParams(final Activity activity, final Site site) {
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Sites/CreateSiteRequest.java]: javadoc

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Tasks/CreateTaskRequest.java [hunk 3]
```diff
@@ -30,9 +30,9 @@ public CreateTaskRequest(final Activity activity, final Task task, HashMap<Strin
                     @Override
                     public void onResponse(JSONObject jsonObject) {
                         try {
-                            String fieldReportJson = jsonObject.get("field_report").toString();
+                            String taskJson = jsonObject.get("task").toString();
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Tasks/CreateTaskRequest.java]: we should make sure this is how the api actually works.

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/MiniSites/MiniSiteFragment.java [hunk 4]
```diff
@@ -63,6 +65,7 @@ public void configureViewWithMiniSite(View view, MiniSite miniSite) {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setHasOptionsMenu(true);
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/MiniSites/MiniSiteFragment.java]: Do we need to do this if we're just gonna use an empty menu? Would it work if we said

<code> setHasOptionsMenu(false); </code>

**Pull request merged**